background/rust.tex

```diff
@@ -11,18 +11,33 @@ \section{The Rust Programming Language} % (fold)
 \subsection{Language Features}
 \label{ssub:rust:features}
 
-This section starts by describing {\rust}'s {\core} library and its type-system, before moving on to presenting some of the language features that make {\rust} so different from C.
+This section starts by describing {\rust}'s standard library and its type-system, before moving on to presenting some of the language features that makes {\rust} so different from {\C}.
 Because {\rust} is a multi-paradigm language it offers a wider range of language constructs that are not present in the procedural paradigm usually found in embedded programming.
 
-\subsubsection{The Core Library}
+\subsubsection{The Rust Standard Library}
+\label{sec:rsl}
 
-\rust's {\core} library, defines the platform independent part of the language.
-This library is not directly accessible to the programmer, instead its stable parts are reexported through {\rust}'s standard library called {\std}.
-In this way, {\core} is not interfaced with directly in normal {\rust} programs, so it can be modified while {\std}'s interface stays the same.
-The standard library also reexports a handful of other libraries and does the task of implementing platform-dependent features for all of its target \glspl{os}.
+The \gls{rsl} is included into all {\rust} programs and strives to be an abstraction layer to write portable applications.
```

Comment:
Vet ikke helt med `... and strives to be an abstraction layer to write...`.
Er std ment som et faktisk "lag med abstraksjoner", eller bare som en "paraply" for Rust's forskjellige bibliotker?